### PR TITLE
Update README.md clarify bluepyefe/efel

### PR DESCRIPTION
**Pull request by anilbey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 A suite to handle single-cell electrophysiological data and to build and validate detailed electrical models.
 
-- [eFEL](#efel) — Electrophys Feature Extraction Library
-- [BluePyEfe](#bluepyefe) — Blue Brain Python E-feature extraction
+- [eFEL](#efel) — Electrophys Feature (E-Feature) Extraction Library
+- [BluePyEfe](#bluepyefe) — E-Feature Extraction from batches of recordings
```
Comment (AurelienJaquier) [inline README.md]: Maybe keep the `Blue Brain Python` part so that it matches the name 'BluePy' part of BluePyEfe

**Comment by darshanmandge** [comment]
I think [emodel-generelisation](https://github.com/BlueBrain/emodel-generalisation) could also be added to the list. What do you think @arnaudon ?

File: README.md (hunk 1 above)
Comment (anilbey) [reply README.md]: Updated.

**Comment by arnaudon** [comment]
sure!

**Comment by anilbey** [comment]
Cool, let's merge this if the updated efel/bluepyefe descriptions are good and @arnaudon can add the new software on another PR when he has time.

**Review by ilkilic — APPROVED** [review]

**Review by AurelienJaquier — APPROVED** [review]
thank you!

**Pull request merged**